apps/builder/app/canvas/use-canvas-shortcuts.ts

```diff
@@ -2,7 +2,10 @@ import { useHotkeys } from "react-hotkeys-hook";
 import { shortcuts, options } from "~/shared/shortcuts";
```

Comment:
Let's call this just canvas-shortcuts.ts in case we get rid of react package here